src/bika/lims/__init__.py

```diff
@@ -46,109 +46,109 @@
 
 # Implicit module imports used by others
 # XXX Refactor these dependencies to explicit imports!
-from bika.lims.config import *  # noqa
```

Comment:
my motivation behind removing `# noqa`:

- it's redundant if we use the global flake8 config
- it interferes with automated code formatting for long lines (different tools have different opinions on which line the `# noqa` needs to be)
- `# noqa` without any addition disables **all** warnings for that line not only the one we want to be ignored

VSCode has flake8 integration but I'm not sure about PyCharm. There is an [open feature request](https://youtrack.jetbrains.com/issue/PY-41427/Support-for-flake8-integration) but also several workarounds seem to exist.

Reply:
one of the workarounds for PyCharm I came across: https://gist.github.com/jsmits/9033655

Reply:
> one of the workarounds for PyCharm I came across: https://gist.github.com/jsmits/9033655

Yep, worked perfectly